drivers/spi/spi_smartbond.c

```diff
@@ -168,11 +584,6 @@ static int spi_smartbond_configure(const struct spi_smartbond_cfg *cfg,
 		return -ENOTSUP;
 	}
 
-	if (spi_cfg->operation & SPI_MODE_LOOP) {
```

Comment:
Why removing this? (It does not seem to belong to what is being done in this commit.)

Reply:
Loop operations should now be supported. Should I create a separate commit for this one?

Reply:
Either another commit or describe in the commit message why it is now available.

That's actually the information I am missing. it's now supported but how? because of DMA or ?
The loop mode is a test mode where a controller can send data to itself, internally, without requiring to short physically MISO/MOSI lines.

Reply:
I was under the impression the loop mode can also reflect physical shorting of MISO/MOSI.